### DISCUSSION TITLE
.014 beta?

### DISCUSSION THREAD

**Discussion by telrod11** [d4rken-org/sdmaid-se#680] [General]
Trying to install ver 014 over ver 013, and getting "App not installed as package conflicts with and existing package".

Is the beta path an uninstall / reinstall, or should it upgrade?

Thanks
![Screenshot_20231002_062634_Settings](https://github.com/d4rken-org/sdmaid-se/assets/105058921/a2dbd78e-b72b-460b-9ff4-e6d986f33ee6)
![Screenshot_20231002_062442_Package installer](https://github.com/d4rken-org/sdmaid-se/assets/105058921/429346b7-3c39-473d-8146-61be089c6cdf)

**Comment by d4rken** [comment]
The GitHub APK can not be installed over the Google Play APK and vice versa.

To switch between the variants, you have to reinstall.

The Google Play APK for 0.14.0 is still in review.

**Comment by telrod11** [comment]
Ah, thank you!!
